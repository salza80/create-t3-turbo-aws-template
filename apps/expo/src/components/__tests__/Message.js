import React from 'react';
import { create } from 'react-test-renderer';

import Message from '../Message';

describe('Message component', () => {
  it('should render without issues', () => {
    const component = create(<Message />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
