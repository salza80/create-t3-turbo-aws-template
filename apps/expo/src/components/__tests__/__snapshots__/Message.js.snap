// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message component should render without issues 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "flexDirection": "row",
      "justifyContent": "flex-start",
      "paddingHorizontal": 10,
      "width": 650,
    }
  }
>
  <Image
    style={
      Object {
        "borderRadius": 30,
        "height": 60,
        "marginRight": 20,
        "marginVertical": 15,
        "width": 60,
      }
    }
  />
  <View>
    <Text />
    <Text
      style={
        Object {
          "color": "#757E90",
          "fontSize": 12,
          "paddingTop": 5,
        }
      }
    />
  </View>
</View>
`;
